web/gds-ui/src/components/ivms101/Address.js

```diff
@@ -42,7 +44,7 @@ const Address = ({index, address, onChange, onDelete}) => {
 
   return (
     <>
-    <Form.Label>Address {index+1}</Form.Label>
+    <Form.Label><Trans>Address {index+1}</Trans></Form.Label>
```

Comment:
This may be tricky … it's rendered as `Address 1` or `Address 2` - should the `<Trans>` tag include the `{index+1}`? How would that show up for the translator?

Reply:
Oops, this will probably not work with Trans. Both cases will be 
```
#: src/components/ivms101/Address.js:47
msgid "Address {0}"
msgstr "添加翻译"
```
in view of the translator. We might need to flatten the code a little bit to work with the translations.